src/NServiceBus.NHibernate.AcceptanceTests-Oracle/ConfigureNHibernatePersistence.cs

```diff
@@ -13,11 +13,11 @@ public void Configure(BusConfiguration config)
     {
         NHibernateSettingRetriever.AppSettings = () => new NameValueCollection
         {
-            {"NServiceBus/Persistence/NHibernate/connection.driver_class", "NHibernate.Driver.OracleDataClientDriver"},
+            {"NServiceBus/Persistence/NHibernate/connection.driver_class", "NHibernate.Driver.OracleManagedDataClientDriver"},
```

Comment:
Is there a specific reason for changing to another oracle driver? The reason is missing in the commit.